### DISCUSSION TITLE
Error: Datasource.Execute failed: repository does not exist

### DISCUSSION THREAD

**Discussion by redfishstix** [vmware/packer-examples-for-vsphere#776] [Help] [answered]
Hi there,

I'm not sure what i'm missing, but I've downloaded the latest release today and when running the build.sh with Ubuntu 22.04 LTS server, I get this error.

`Error: Datasource.Execute failed: repository does not exist
on /opt/packer/packer-examples-for-vsphere-develop/builds/linux/ubuntu/22-04-lts/linux-ubuntu.pkr.hcl line 34:(source code not available)`
`Error: Unsupported attribute
on /opt/packer/packer-examples-for-vsphere-develop/builds/linux/ubuntu/22-04-lts/linux-ubuntu.pkr.hcl line 61:
(source code not available)
This object does not have an attribute named "build_version".
Error: Unsupported attribute
on /opt/packer/packer-examples-for-vsphere-develop/builds/linux/ubuntu/22-04-lts/linux-ubuntu.pkr.hcl line 43:
(source code not available)
This object does not have an attribute named "build_version".
Error: Unsupported attribute
on /opt/packer/packer-examples-for-vsphere-develop/builds/linux/ubuntu/22-04-lts/linux-ubuntu.pkr.hcl line 42:
(source code not available)
This object does not have an attribute named "git-repository".
Error: Unsupported attribute
on /opt/packer/packer-examples-for-vsphere-develop/builds/linux/ubuntu/22-04-lts/linux-ubuntu.pkr.hcl line 48:
(source code not available)
This object does not have an attribute named "vm_name".`

Any help would be greatly appreciated!
Thanks!

**Comment by tenthirtyam** [comment, marked as answer]
Did you just download a zip from develop branch or follow the steps in the documentation to clone or download and git init?

**Reply by tenthirtyam** [reply]
https://vmware-samples.github.io/packer-examples-for-vsphere/getting-started/download/#download

**Reply by redfishstix** [reply]
Yup, that was it. I downloaded the zip instead of following my instructions :\ https://vmware-samples.github.io/packer-examples-for-vsphere/getting-started/download/
Thanks and sorry for that.
